browser/ui/webui/new_tab_page/brave_new_tab_ui.cc

```diff
@@ -58,15 +59,16 @@ using ntp_background_images::NTPCustomImagesSource;
 
 namespace {
 
-std::string GetSearchWidgetDefaultHost(PrefService* local_state) {
+std::string GetSearchWidgetDefaultHost(
+    regional_capabilities::RegionalCapabilitiesService* regional_capabilities) {
   constexpr char kBraveSearchHost[] = "search.brave.com";
   constexpr char kYahooSearchHost[] = "search.yahoo.co.jp";
-  if (!local_state) {
-    CHECK_IS_TEST();
```

Comment:
RegionalCapabilitiesService is available in tests? I think some of these are triggered by upstream browser tests

Reply:
We should be able to add tests for this now when we're ready

Code didn't have tests yet - so I didn't change that

Reply:
this wasn't here for tests for this code, it was here for other tests where this code gets loaded. I guess if tests are passing in CI then it should be ok